docs/references/kontena-yml.md

```diff
@@ -52,17 +52,28 @@ services:
 
 #### expose
 
-Expose a service from stack for use by other stacks. Read more about exposing services [here](https://kontena.io/docs/using-kontena/stacks#exposing-services-between-stacks).
+Expose a named service from stack for use by other stacks. Read more about exposing services [here](https://kontena.io/docs/using-kontena/stacks#exposing-services-between-stacks).
 
 ```
 expose: api
+```
+
+#### services
+
+The named services that belong to the stack. See the [Service Configuration section](#service-configuration-reference) for the keys usable for each service.
+
+```
 services:
   api:
     image: registry/api:latest
   db:
     image: mariadb:latest
 ```
 
+The service name must start with a letter, number or underscore, and may also contain dashes.
+
+There is no limitation on the length of a service name per se, but the resulting service container FQDN (`service-1.stack-name.grid-name.kontena.local`) must be 64 bytes or less. The effective service name length maximum thus also depends on the grid and stack names, so they should be kept reasonably short.
```

Comment:
There isn't really any issue to link to for it, though... probably need to create one first?